tests/e2e/src/main.rs

```diff
@@ -0,0 +1,219 @@
+mod config;
+
+use config::Config;
+use log::info;
+use sp_core::crypto::Ss58Codec;
+use sp_core::{sr25519, Pair};
+use sp_runtime::{generic, traits::BlakeTwo256};
+use std::collections::HashSet;
```

Comment:
Have you run fmt & clippy? Or maybe they don't check excluded crates?

Reply:
its clean, but we could introduce a workflow that lints auxiliary code. I made a ticket : https://cardinal-cryptography.atlassian.net/browse/AZ-246